F/FlowCutterPACE17/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FlowCutterPACE17"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kit-algo/flow-cutter-pace17.git", "fd405a2b003c9a93838e0488728addc0610a769f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flow-cutter-pace17/
+$CXX -c -Wall -std=c++11 -O3 -DNDEBUG src/*.cpp
+$CXX -o flow_cutter_pace17 *.o
+install_license LICENSE
+mkdir $bindir 
+mv flow_cutter_pace17 $bindir/flow_cutter_pace17
```

Comment:
A bit simpler:
```suggestion
mkdir -p "${bindir}"
$CXX -o "${bindir}/flow_cutter_pace17${exeext}" *.o
install_license LICENSE
```
:wink: